### PR TITLE
ARGO-200 Implement Endpoint status timeline request

### PR DESCRIPTION
**Pull request by kaggis**
- Implement model, view, controller
- Add unit tests

### CODE REVIEW

**Comment by kaggis** [comment]
please review 
/cc @makistsantekidis @pkoro

File: doc/api/status-results.md [hunk 1]
```diff
@@ -148,40 +146,123 @@ Reponse body:
...
+#### Response Code
+```
+Status: 200 OK
+```
 
-Headers: `Status: 200 OK`
+### Response body
 
-#### Response body
+##### List All metrics:
```
Comment (pkoro) [inline doc/api/status-results.md]: s/metrics/endpoints/

File: doc/api/status-results.md [hunk 2]
```diff
@@ -148,40 +146,123 @@ Reponse body:
...
+				<status timestamp="2015-05-01T01:00:00Z" status="CRITICAL"></status>
+				<status timestamp="2015-05-01T02:00:00Z" status="OK"></status>
+			</group>
+		</group>
+	</group>
+</root>
+```
+
+
+#### List specific metric (`hostname=cream01.afroditi.gr`):
```
Comment (pkoro) [inline doc/api/status-results.md]: @kaggis it should be List specific endpoint here. And also it may be better to remove the example from the header.

Comment (kaggis) [inline doc/api/status-results.md]: Yes indeed, will fix it

**Comment by pkoro** [comment]
Other than my two comments in `doc` part +1

**Pull request merged**